app/helpers/file_helper.rb

```diff
@@ -18,8 +18,8 @@ def accept_file(file, name, kind)
       accept = ['image/png', 'image/gif', 'image/bmp', 'image/tiff', 'image/jpeg', 'image/x-ms-bmp']
     when 'code'
       accept = ['text/x-pascal', 'text/x-c', 'text/x-c++', 'text/plain', 'text/', 'application/javascript, text/html',
```

Comment:
Can we also fix this here as well?
'application/javascript, text/html' --> missing ' in the middle...